runtime/pangoro/src/pallets/sudo.rs

```diff
@@ -22,4 +22,5 @@ use crate::*;
 impl pallet_sudo::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = pallet_sudo::weights::SubstrateWeight<Self>;
```

Comment:
Why not benchmarking based on our runtime?  See https://github.com/paritytech/polkadot/pull/7118

Reply:
Sudo is only used in testnets.